document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/exception/HandlerException.java

```diff
@@ -1,11 +1,29 @@
 package uk.gov.companieshouse.document.generator.accounts.exception;
 
+/**
```

Comment:
Several points:
'The class...' is superfluous. This is a class doc. Same applies to 'is a form of exception' as that is in the name.
How about:L
Wrapper exception that hides lower level exceptions from the caller.

That said, it is still not clear either from this Javadoc or from the method Javadocs what you should do/are allowed to do if you see this exception. It needs to be clear in one of these places, and I suspect it is the method docs.

Reply:
Same points apply to ServiceException too.
Also, no need for the link as you are on the page that it links to. Furthermore, in any Javadocs where type is there in the return or a parameter, a link will be generated and you do not need to make another one right next to it, as it just clutters the text.
If you look at the method Javadcos here, you'll see what I mean:
https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html

Reply:
I have altered the text and have pushed it but left the one last just to show the reason for why we have the custom exception.

Reply:
Additionally, I have only now just figured out the magic of ctrl + J in intelliJ where you can view the JD when it gets rendered. So am slowly going through correcting them :P

Reply:
Following a chat elsewhere, I take back some of my comments about instructing the caller on what to do. That point can sometimes apply, but not here.
Specifically, I was applying a previous lesson from a stateful service that threw exceptions in situations that sometimes implied that the method failed, but you still had a session, or sometimes the session had gone, or sometimes the service was generally somewhere up a creek... and the API could have made all this clear to the client but did not. None of that applies here: the method simply failed!

Reply:
Ahh I see, nah I don't think this applies here, it may be hard to see currently what to do, but it will make more sense when the refined exception handling has been done as there will be exceptions for certain things. Additionally, the design will be updated soon to take into account what the accounts implementation will do. Which in simple terms would be the following:

ServiceException throw -> caught in Handler layer, logs service exception details, throws new HandlerException -> caught in controller layer, logs the handlerexception, throws controller exception etc.